### DISCUSSION TITLE
Receiving the probability map

### DISCUSSION THREAD

**Discussion by adigol** [ultralytics/yolov5#5276] [General]
Is it possible to receive the probability map of each pixel in the image(or in any bounding box) like in the attached images?
![1_3RgY2fGUdJKrFv2MI53nxg](https://user-images.githubusercontent.com/35373818/138244670-8c717fa1-a332-46ce-897c-e2cea8b41921.png)
![1_m8p5lhWdFDdapEFa2zUtIA](https://user-images.githubusercontent.com/35373818/138246321-65a8ff3c-df6e-4df5-a12e-6977e5b844a8.jpeg)

**Comment by glenn-jocher** [comment]
@adigol this information is contained in the Detect() forward method feature maps here. You'd have to index these as appropriate to extract the values you are interested in:
https://github.com/ultralytics/yolov5/blob/15e8c4c15bff0e3c729868e2c31f80f918e31fb2/models/yolo.py#L36-L74

**Reply by DoSquared** [reply]
> @adigol this information is contained in the Detect() forward method feature maps here. You'd have to index these as appropriate to extract the values you are interested in:
> 
> https://github.com/ultralytics/yolov5/blob/15e8c4c15bff0e3c729868e2c31f80f918e31fb2/models/yolo.py#L36-L74

Can you please elaborate more?

**Comment by DoSquared** [comment]
I have the same question

**Reply by amartyacodes** [reply]
Hi, do you have the information regarding how to obtain it? I would be grateful if you can share
